docs/testing.md

```diff
@@ -0,0 +1,60 @@
+# Testing
+
+EXTPAR is tested with an adapted version of the [COSMO technical testsuite :material-open-in-new:](https://github.com/C2SM-RCM/testsuite){:target="_blank"}.
```

Comment:
Should this repo also be moved to C2SM org?

Reply:
Hm not really, in extpar we use a copy of it.
And no one really uses it anymore.